src/screens/CreatePost/index.js

```diff
@@ -610,6 +622,7 @@ const CreatePost = () => {
 
   const isPollButtonDisabled = () => getReducedPoll().length < 2;
   const isButtonDisabled = () => {
+    if (loadingPost) return true;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The modification to the `isButtonDisabled` function to include a check for `loadingPost` is a good practice. It prevents the user from initiating another post creation while one is already in progress, thus preventing potential data inconsistencies or errors.
```diff
+    if (loadingPost) return true;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->